### ISSUE TITLE
pod spec lint error

### ISSUE THREAD

**Issue by iblacksun** [CocoaPods/CocoaPods#5848]
# Report

## What did you do?

ℹ Please replace this with what you did.  
Run `pod spec lint xxx.podspec --verbose`

## What did you expect to happen?

e.g. podspec lint successful

## What happened instead?

Check dependencies
“Use Legacy Swift Language Version” (SWIFT_VERSION) is required to be configured correctly for targets which use Swift. Use the [Edit > Convert > To Current Swift Syntax…] menu to choose a Swift version or use the Build Settings editor to configure the build setting directly.

** BUILD FAILED **
The following build commands failed:
	Check dependencies
(1 failure)
 -> XXX (2.0.2)
    - ERROR | xcodebuild: Returned an unsuccessful exit code.

Analyzed 1 podspec.

[!] The spec did not pass validation, due to 1 error.

### Stack

```
   CocoaPods : 1.1.0.rc.1
        Ruby : ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin15]
    RubyGems : 2.6.6
        Host : Mac OS X 10.12 (16A319)
       Xcode : 8.0 (8A218a)
         Git : git version 2.10.0
Ruby lib dir : /Users/iblacksun/.rvm/rubies/ruby-2.3.0/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 5df705e4da8e4c18d885f42456b9646cfb5ecacc
```

### Installation Source

```
Executable Path: /Users/iblacksun/.rvm/gems/ruby-2.3.0/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.1.0
```

**Comment by OskarGroth** [comment]
I'm seeing the exact same issue. Setting the `Use Legacy Swift Language Version` flag to `No` does not resolve the issue.

**Comment by benasher44** [comment]
This is fixed in #5841 and will be available in an upcoming release shortly.
